Modules/HRV_DFA/DFA.cpp

```diff
@@ -0,0 +1,231 @@
+#include "DFA.h"
+
+
+void Polyfit::createNarray()
+{
+    int m_Size = m_End-m_Start+1;
+    m_N.resize(m_Size);
+    std::iota(m_N.begin(), m_N.end(), m_Start);
+}
+
+void Polyfit::readInput()
+{
+    // this only for debugging
```

Comment:
Usuwamy elementy potrzebne do debugowania

Reply:
Tak jak było wspomniane to zostało dlatego, że nie wiedziałam, czy jeszcze coś mam zmieniać w pozostałym kodzie, wtedy muszę sprawdzić, czy wszystko liczy się poprawnie

Reply:
done